scene/gui/graph_node.cpp

```diff
@@ -559,7 +559,15 @@ Color GraphNode::get_connection_output_color(int p_idx) {
 void GraphNode::_input_event(const InputEvent& p_ev) {
 
 	if (p_ev.type==InputEvent::MOUSE_BUTTON) {
-		get_parent_control()->grab_focus();
+
+		ERR_EXPLAIN("GraphNode must be the child of a GraphEdit node.");
+		ERR_FAIL_COND(get_parent_control() == NULL);
+
+		if (get_parent_control())
```

Comment:
As far as I understand ``ERR_FAIL_COND`` already returns if its test is true, so you don't need the `if/else` afterwards.